### PR TITLE
Add Weave Cloud agents

### PR DESCRIPTION
**Pull request by errordeveloper**

### CODE REVIEW

File: kubernetes-templates/weave-cloud/0/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.catalog:
+  name: "Weave Cloud Suite"
```
Comment (lukemarsden) [inline kubernetes-templates/weave-cloud/0/rancher-compose.yml]: I think "Tools" is a more appropriate word than "Suite".

File: kubernetes-templates/weave-cloud/0/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  description: "All you need to implement Continuous Delivery with hosted Prometheus monitoring and visual distributed system analysis/debug tools"
```
Comment (lukemarsden) [inline kubernetes-templates/weave-cloud/0/rancher-compose.yml]: Maybe change:

"All you need to implement Continuous Delivery with hosted Prometheus monitoring and visual distributed system analysis/debug tools"

"Weave Cloud is a hosted add-on to Kubernetes which provides Continuous Delivery, Prometheus for Monitoring and a visual dashboard for troubleshooting & debugging microservices. This package contains the agents which connect your cluster to Weave Cloud. You will need a service token which you can get from https://cloud.weave.works/"

File: kubernetes-templates/weave-cloud/0/rancher-compose.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  version: "latest"
+  questions:
+    - variable: "WEAVE_CLOUD_SERVICE_TOKEN"
+      description: "Service authentication toke for you Weave Cloud instance, signup at https://cloud.weave.works"
```
Comment (lukemarsden) [inline kubernetes-templates/weave-cloud/0/rancher-compose.yml]: s/toke/token

s/signup/get one

File: kubernetes-templates/weave-cloud/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Weave Cloud Suite
+
+Weave Cloud provides all you need to implement Continuous Delivery with hosted Prometheus monitoring and visual distributed system analysis/debug tools.
```
Comment (lukemarsden) [inline kubernetes-templates/weave-cloud/README.md]: => "Weave Cloud is a hosted add-on to Kubernetes which provides Continuous Delivery, Prometheus for Monitoring and a visual dashboard for troubleshooting & debugging microservices"

File: kubernetes-templates/weave-cloud/README.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+_To learn more and singup please visit [weave.works](https://weave.works)._
```
Comment (lukemarsden) [inline kubernetes-templates/weave-cloud/README.md]: s/singup/sign up

File: kubernetes-templates/weave-cloud/README.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+You will need to provide an authentication token before you install this stack.
```
Comment (lukemarsden) [inline kubernetes-templates/weave-cloud/README.md]: You will need a service token which you can get from https://cloud.weave.works/

File: kubernetes-templates/weave-cloud/config.yml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+name: "Weave Cloud Suite"
+description: |
+  All you need to implement Continuous Delivery with hosted Prometheus monitoring and visual distributed system analysis/debug tools
```
Comment (lukemarsden) [inline kubernetes-templates/weave-cloud/config.yml]: "Weave Cloud is a hosted add-on to Kubernetes which provides Continuous Delivery, Prometheus Monitoring and a visual dashboard for troubleshooting & debugging microservices. This package contains the agents which connect your cluster to Weave Cloud. You will need a service token which you can get from https://cloud.weave.works/"

File: kubernetes-templates/weave-cloud/config.yml [hunk 8]
```diff
@@ -0,0 +1,6 @@
...
+version: "latest"
+category: Continuous Delivery, Monitoring & Visualization
```
Comment (lukemarsden) [inline kubernetes-templates/weave-cloud/config.yml]: s/Visualization/Troubleshooting

**Comment by rawmind0** [comment]
Hi @errordeveloper ...Thanks so much for the cool PR .... :)

I've tested the package and is working ok. Please take a look to the @lukemarsden suggestions before merge. 

Thanks..best regards...

**Comment by errordeveloper** [comment]
I've address the comments. Thanks @rawmind0 and @lukemarsden, PTAL.

File: kubernetes-templates/weave-cloud/README.md (hunk 5 above)
Comment (rawmind0) [inline kubernetes-templates/weave-cloud/README.md]: This is not addressed correctly. 
s/sing up/sign up/

**Comment by rawmind0** [comment]
Hi @errordeveloper ... Thanks for address the comments. There is one still wrong, please review y comment. 

Once of this will be addressed, it's ok.. :)

File: kubernetes-templates/weave-cloud/README.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
+# Weave Cloud Agents
+
+Weave Cloud is a add-on to Kubernetes which provides Continuous Delivery, along with hosted Prometheus Monitoring and a visual dashboard for exploring & debugging microservices.
+
+This package contains the agents which connect your cluster to Weave Cloud.
+
+_To learn more and sing up please visit [Weaveworks website](https://weave.works)._
```
Comment (lukemarsden) [inline kubernetes-templates/weave-cloud/README.md]: sign

**Comment by lukemarsden** [comment]
LGTM!

**Comment by errordeveloper** [comment]
I've rebased and fixed the type @lukemarsden pointed out. How's it looking to you now @rawmind0? I'm really unsure which comment you were referring to...

File: kubernetes-templates/weave-cloud/README.md (hunk 9 above)
Comment (rawmind0) [inline kubernetes-templates/weave-cloud/README.md]: Just change the line 7 from
+_To learn more and sing up please visit [Weaveworks website]
to
+_To learn more and sign up please visit [Weaveworks website]

**Comment by rawmind0** [comment]
Hi @errordeveloper ...

the package is working well. Just correct the typo commented and i'll merge it. :)

The typo is in file  kubernetes-templates/weave-cloud/README.md line 7....

**Comment by errordeveloper** [comment]
@rawmind0 haha, I've changed it locally and haven't amended the commit – done, thank you!

**Comment by rawmind0** [comment]
Thanks... @errordeveloper 

LGTM

**Pull request merged**